docs/sources/panels/visualizations/bar-chart.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Bar chart"
+description = "Bar chart visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
```

Comment:
Please add a weight that will maintain alphabetical order.

Reply:
I was wondering if we remove all weights won't it be in alphabetical order?

Reply:
From all the topics with weights, yes.